Bottom: 353e7a0e8dbd7d5c1c019682e7579f052e3c5849
Top:    064ba81ee32a906530fa98bbfdc6abc33f68241f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-13 15:18:30 +0800

Refresh of virtio-net-map-offload

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 52be5b3..b5c011a 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -2620,7 +2620,7 @@ static int virtnet_bpf_ctrl_entry_op(struct bpf_offloaded_map *offmap,
 		return -EFAULT;
 	}
 
-	printk("eBPF op was done successfully!\n");
+	printk("eBPF op was done successfully for op %d!\n", cmd);
 
 	/* Copy outputs */
 	if (out_key)
